src/routes/__error.svelte

```diff
@@ -0,0 +1,54 @@
+<script context="module">
+	export function load({ error, status }) {
+		return {
+			props: { error, status }
+		};
+	}
+</script>
+
+<script>
+	import ButtonLight from '$lib/shared/Button/ButtonLight.svelte';
+	import SubmitButton from '$lib/shared/Button/SubmitButton.svelte';
+
+	/** @type {number} */
+	export let status;
+
+	/** @type {Error} */
+	export let error;
+</script>
+
+{#if status === 404}
```

Comment:
Werkt dus prima als je online bent, maar bij offline (op https://deploy-preview-140--jungle-gym.netlify.app/games/jldfldfjldfjlk) krijg ik een error. Hier rekening mee gehouden?

Reply:
Huh.. dat is raar deze error kreeg ik niet lokaal.. 😓

Reply:
![image](https://user-images.githubusercontent.com/34123512/119948101-ca021380-bf98-11eb-88ca-f3eef9e6e311.png)

Reply:
Dit is gefixt :)